stripecardscan/src/main/java/com/stripe/android/stripecardscan/framework/util/Encode.kt

```diff
@@ -15,11 +20,43 @@ fun b64Encode(b: ByteArray): String =
     Base64.encodeToString(b, Base64.NO_WRAP)
 
 /**
- * Encode a serializable object to a x-www-url-encoded string
+ * Encode a serializable object to a x-www-url-encoded string. The source object must convert to a
+ * [Map] so that the parameters can be named.
  */
-@ExperimentalSerializationApi
 fun <T> encodeToXWWWFormUrl(serializer: SerializationStrategy<T>, value: T): String =
-    QueryStringFactory.create(NetworkConfig.form.encodeToMap(serializer, value))
+    QueryStringFactory.create(NetworkConfig.json.encodeToJsonElement(serializer, value).toMap())
```

Comment:
Rather than using the experimental form encoder, this converts a serializable object to its equivalent JSON, then converts the JSON into a construction of `Map` and `List` with primitive values.